spec/features/idv/flow_spec.rb

```diff
@@ -347,7 +347,9 @@
     end
 
     scenario 'pick USPS address verification' do
-      sign_in_and_2fa_user
+      user = sign_in_and_2fa_user
+      loa3_sp_session
```

Comment:
The two specs aren't _exactly_ the same, as the flow_spec example verifies immediately after the profile is created. I don't think that matters much though, as that scenario is guaranteed to never happen in real life.